edx_event_bus_kafka/internal/producer.py

```diff
@@ -34,6 +34,16 @@
 # CloudEvent standard name for the event type header, see
 # https://github.com/cloudevents/spec/blob/v1.0.1/kafka-protocol-binding.md#325-example
 EVENT_TYPE_HEADER_KEY = "ce_type"
+ERROR_STRING_TEMPLATE = "Error publishing event to event bus. error={error} {context}"
```

Comment:
It makes me mildly nervous to have a format string divorced from the format call that has to define the error/context parameters. (A linter won't notice a mismatch. If we had complete test coverage I guess it would be fine though.) Perhaps it should be a function? It could call logger.error and logger.record_exception.

Reply:
[fixed]